Sources/Sentry/Public/SentrySDK.h

```diff
@@ -301,6 +301,14 @@ SENTRY_NO_INIT
  */
 + (void)crash;
 
+/**
+ * Report to the current ongoing UIViewController transaction
+ * that the screen is fully loaded and ready to be used,
+ * this creates a new span with the amount of time it took
+ * from initializing the view controller until it is ready.
```

Comment:
`m`:
```suggestion
 * Report to the current ongoing UIViewController transaction
 * that the screen content is fully loaded and displayed.
 * This adds a span and measurement to the transaction.
```

Reply:
This don't add a measurement.

Reply:
As discussed yesterday in the daily we should add a measurement for it.